data/ecosystems/x/xdc.toml

```diff
@@ -119,3 +125,82 @@ url = "https://github.com/XDCFoundation/XRC777-SMART-CONTRACT"
 
 [[repo]]
 url = "https://github.com/XDCFoundation/Youtube_Content_Engine"
+
+[[repo]]
+url = "https://github.com/AnilChinchawale/chains"
+
+[[repo]]
+url = "https://github.com/AnilChinchawale/xdc-testnetwork-leo"
+
+[[repo]]
+url = "https://github.com/AnilChinchawale/erigon"
+
+[[repo]]
+url = "https://github.com/AnilChinchawale/chainlist"
```

Comment:
I've checked the commits diff with the forked defillama chainlist repo and I can't see any contribution from XDC, can you point it out to me or remove this repo?